gobblin-utility/src/main/java/org/apache/gobblin/util/HadoopUtils.java

```diff
@@ -145,6 +145,12 @@ public static void deletePath(FileSystem fs, Path f, boolean recursive) throws I
     }
   }
 
+  public static void deleteDirectories(FileSystem fs, List<String> directoriesToDelete, boolean recursive) throws IOException {
+    for (String directory : directoriesToDelete) {
+      deletePath(fs, new Path(directory), recursive);
```

Comment:
We may want to move the data into trash instead of deleting it directly. In the interest of unblocking, please just create a new ticket for this and you should work on it next.